src/db/z_db.erl

```diff
@@ -356,7 +368,7 @@ equery(Sql, Parameters, Context, Timeout) ->
 
 
 %% @doc Insert a new row in a table, use only default values.
-%% @spec insert(Table, Context) -> {ok, Id}
+-spec insert(table_name(), #context{}) -> boolean().
```

Comment:
The spec in line 371 uses a record, please define a type for the record and use that instead.